src/helpers/QueryHelper.ts

```diff
@@ -4,5 +4,5 @@ import { DatabaseCore } from "helpers/Constants";
 export function generateDataviewTableQuery(columns: Record<string, DatabaseColumn>, fromQuery: string): string {
     return `TABLE ${Object.keys(columns)
         .filter((key) => !key.startsWith("__") && !key.endsWith("__"))
-        .join(",")},${DatabaseCore.DATAVIEW_FILE},${DatabaseCore.FRONTMATTER_KEY} ${fromQuery.replace(/['"]+/g, '')}`
+        .join(",")},${DatabaseCore.DATAVIEW_FILE},${DatabaseCore.FRONTMATTER_KEY} ${fromQuery}`
```

Comment:
For the new escaping to work, we have to stop stripping out the quotes and double quotes

Reply:
Let me know when you think its ready to merge